src/utils.ts

```diff
@@ -68,3 +80,26 @@ export function useBase (base: string, handle: PHandle): PHandle {
     return handle(req, res)
   }
 }
+
+export function useQuery (req: IncomingMessage): URLSearchParams {
+  const url = new URL(req.url || '', 'http://localhost')
+  return url.searchParams
```

Comment:
I am not found of the `searchParams` object honesty, quite hard to loop on it, I would prefer the object syntax if possible, cc @pi0

Reply:
Agreed. We might rename current utility to `useParams`/`getParams` and a new `useQuery`/`getQuery` and both may be moved to ufo (only exporting request wrapper from here)

Reply:
Just created `getQuery` util to return object.
For the sake of consistency, I do use `get` prefix for all utils

Reply:
It was just a suggestion for query or utils... Since for body we actually read body. For consistency use* sounds better